modules/Bio/Otter/Auth/Access.pm

```diff
@@ -22,11 +22,35 @@ use warnings;
 use Try::Tiny;
 use Carp;
 use Scalar::Util 'weaken';
+use Crypt::JWT qw(decode_jwt);
```

Comment:
This commit fails on travis tests, can't be merged like this.

Reply:
The tests will have to include Crypt::JWT.